util/network-devp2p/src/discovery.rs

```diff
@@ -522,7 +533,8 @@ impl Discovery {
 		let ping_to = NodeEndpoint::from_rlp(&rlp.at(2)?)?;
 		let timestamp: u64 = rlp.val_at(3)?;
 		self.check_timestamp(timestamp)?;
-		let mut response = RlpStream::new_list(3);
+		// let enr_seq = rlp.val_at::<u64>(4).ok();
```

Comment:
```suggestion
```

Stray code?

Reply:
I actually left it on purpose for the time when we actually start using peers' ENRs.